precompiles/warden/events.go

```diff
@@ -417,7 +417,7 @@ func (p Precompile) GetUpdateKeychainEvent(ctx sdk.Context, _ *common.Address, e
 	var marshaled []byte
 	if err := precommon.ParseSdkEvent(eventUpdateKeychain, func(m proto.Message) {
 		marshaled, err = proto.Marshal(m)
-		typedEvent.Unmarshal(marshaled)
+		typedEvent.Unmarshal(marshaled) //nolint:errcheck
```

Comment:
Can we change `ParseSdkEvent` to return error from callback and check err in this row?

Reply:
+